components/HospitalCard.js

```diff
@@ -41,11 +59,15 @@ export default function HospitalCard(props) {
             w={['24', '28']}
             p={2}
             textAlign="center"
-            bgColor={hospital.available_bed > 0 ? 'gray.200' : 'red.200'}
+            bgColor={
+              hospital.available_bed > 0
+                ? bgDistanceAvailable
+                : bgDistanceNotAvailable
+            }
             borderRadius="md"
           >
             <Text fontSize={['sm', 'md']} fontWeight="bold">
-              {hospital.distance.toFixed(2)} KM
+              {hospital.distance?.toFixed(2)} KM
```

Comment:
i'm adding this because if we didn't grant location access, this cause an error

Reply:
Oh! Wait checking